.github/workflows/test_arm64.yaml

```diff
@@ -0,0 +1,39 @@
+name: Test ARM64
+on: [push, pull_request]
+jobs:
+  run:
+    runs-on: actuated-arm64-8cpu-32gb
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - linux-arm64-unit-test-1-cpu
+          - linux-arm64-unit-test-2-cpu
+          - linux-arm64-unit-test-4-cpu
+          - linux-arm64-unit-test-4-cpu-race
```

Comment:
Based on the existing tests in https://github.com/etcd-io/raft/blob/main/.github/workflows/test.yaml#L10-L11 I think we only need to enable `linux-arm64-unit-4-cpu-race`